influxdb3_write/src/write_buffer/persister.rs

```diff
@@ -848,13 +857,13 @@ mod tests {
             .write()
             .close_segment(first_segment_range.start_time)
             .unwrap();
-        let persisted = closed_segment
+        let _persisted = closed_segment
```

Comment:
I think you can just change this to
```suggestion
        closed_segment
```

unless you plan on using this later?

Reply:
haha yes, leftover cruft, thanks 😄